src/submission/validation.ts

```diff
@@ -69,6 +72,9 @@ export const validateRegistrationData = async (
     }
   }
 
+  // catch any rows that are exactly the same
+  errors = errors.concat(checkUniqueRecords(ClinicalEntityType.REGISTRATION, newRecords, true));
```

Comment:
so I'm now a bit confused, it's smart to pick sample submitter id as unique Id for a record in registration, so why do we still need the `useAllRecordsValues` ?

Reply:
yea I have removed sample_registration from this check, we can check it in `conflictingNewSample` and we can use `NEW_SAMPLE_ID_CONFLICT` (I checked it with Rosi)